src/main/webapp/script.js

```diff
@@ -10,13 +13,19 @@ function initIndex() {
     retrieveEvents();
 }
 
-function retrieveEvents () {
-  fetch('/retrieve-events').then(response => response.json()).then((events) => {
-    const eventsList = document.getElementById('event-list');
-    events.forEach((event) => {
-        eventsList.appendChild(createEventElement(event));
-    })
-  });
+async function retrieveEvents () {
+    // Retreives events.
+    fetch('/retrieve-events').then(response => response.json()).then((events) => {
+        const eventsList = document.getElementById('event-list');
+        events.forEach((event) => {
+            eventsList.appendChild(createEventElement(event));
+                new google.maps.Marker({
+                position: {lat: event.address.latitude, lng: event.address.longitude},
```

Comment:
No need to make this change, just an FYI, but I think in real production code, you'd want to make sure we do have lat/lng here. There's a chance we fail to get lat/lng when creating the event, in which case this would fail. I think if we don't have it, it'll default to zero since it's a double, so we'll get misplaced markers. But then having a marker at lat/long 0/0 is valid, so it's a little tricky to verify